node/narwhal/src/primary.rs

```diff
@@ -1079,7 +1079,7 @@ mod tests {
             let mut members = IndexMap::new();
 
             for i in 0..COMMITTEE_SIZE {
-                let socket_addr = format!("127.0.0.1:{i}").parse().unwrap();
+                let socket_addr = format!("127.0.0.1:500{i}").parse().unwrap();
```

Comment:
what if we have a committee size greater than 10 :detective:?

Reply:
I thought that might be unlikely in tests—fixed just in case!